hilti/toolchain/include/ast/node.h

```diff
@@ -441,6 +506,7 @@ class Node {
 
         if ( auto i = std::find(_children.begin(), _children.end(), n); i != _children.end() ) {
             (*i)->_parent = nullptr;
```

Comment:
It seems the update to `_parent` shouldn't matter since the node will be destroyed, suggest to drop.

Also below.

Reply:
See other comment.